interfaces/PA-App/src/app/personal-components/login-identity/login-identity.component.ts

```diff
@@ -37,6 +37,7 @@ export class LoginIdentityComponent extends PersonalComponent {
         this.isInProgress = true;
         this.conversationService.startLoading();
         // Here goes something that retrieves up-to-date conversation-history??
+        this.paData.setLoggedIn();
```

Comment:
I think the `PaDataService` will handle this itself. (so 'inside' `this.paData.login()`.